crucible/src/Lang/Crucible/Analysis/Postdom.hs

```diff
@@ -86,15 +91,19 @@ postdomMap m = r
           , let Just pd_id = Map.lookup pd idMap
           ]
 
-postdomAssignment :: forall ext blocks ret . BlockMap ext blocks ret -> CFGPostdom blocks
-postdomAssignment m = fmapFC go m
-  where pd = postdomMap m
+postdomAssignment :: forall ext blocks ret
+                   . BlockMap ext blocks ret
+                  -> [Some (BlockID blocks)]
+                  -> CFGPostdom blocks
+postdomAssignment m breakpointIds = fmapFC go m
+  where pd = postdomMap m breakpointIds
         go :: Block ext blocks ret c -> Const [Some (BlockID blocks)] c
         go b = Const $ fromMaybe [] (Map.lookup (Some (blockID b)) pd)
 
 -- | Compute posstdom information for CFG.
 postdomInfo :: CFG ext b i r -> CFGPostdom b
-postdomInfo g = postdomAssignment (cfgBlockMap g)
+postdomInfo g =
+  postdomAssignment (cfgBlockMap g) (Bimap.keysR $ cfgBreakpoints g)
```

Comment:
As we discussed, I think this is the wrong default... we ought to be performing the post-dominator analysis normally as though we are simply going to step through breakpoints, and only pass in the ones here that we are actually going to use an execution feature to break-and-return from.